tests/c/api_test.c

```diff
@@ -596,12 +596,22 @@ static int do_cases(void)
   CHK1(test_so, "libtaos_odbc.dylib", 0);
 #elif defined(_WIN32)
   CHK1(test_so, "taos_odbc.dll", -1);
+
+  char dll_fullname[256] = {0};
 #ifdef TODBC_X86
-  CHK1(test_so, "C:/TDengine/taos_odbc/x86/bin/taos_odbc.dll", 0);
+  char *dll_fullpath = "C:/TDengine/taos_odbc/x86/bin";
 #else
-  CHK1(test_so, "C:/TDengine/taos_odbc/x64/bin/taos_odbc.dll", 0);
+  char *dll_fullpath = "C:/TDengine/taos_odbc/x64/bin";
 #endif
-  CHK1(test_so, "taos_odbc.dll", -1);
+  (void)snprintf(dll_fullname, sizeof(dll_fullname), "%s/%s", dll_fullpath, "taos_odbc.dll");
+
+  if (!SetDllDirectory(dll_fullpath)) {  
```

Comment:
still not prefer such `hard-coded` bounding approach

Reply:
You are right，I will try to separate the path selection between compile installation and package installation